pvlib/ivtools/sdm.py

```diff
@@ -564,25 +564,32 @@ def fit_desoto_sandia(ivcurves, specs, const=None, maxiter=5, eps1=1.e-3):
         I_L_ref : float
             light current at STC [A]
```

Comment:
```suggestion
            Light current at STC [A]
```

May as well do this line also